### DISCUSSION TITLE
What does the value of `TouchSensor` mean?

### DISCUSSION THREAD

**Discussion by MeetWq** [cyberbotics/webots#6535] [General]
I created a cube with a mass of `1kg` and placed a `TouchSensor` of type `force` below it.
The value of the touch sensor was around `98`, what's the unit of the value?

![2024-04-10_14-37-50](https://github.com/cyberbotics/webots/assets/33149974/04102d30-993e-4a78-8ac6-02b45f5bcb47)

I changed the type of the `TouchSensor` to `force-3d`, the values are even more confusing.

![2024-04-10_14-48-57](https://github.com/cyberbotics/webots/assets/33149974/10b59859-830b-4635-afcf-ae1450eee7a8)

Here is the webots project I used:

[test.zip](https://github.com/cyberbotics/webots/files/14928976/test.zip)

**Comment by MeetWq** [comment]
@omichel

**Reply by ShuffleWire** [reply]
For the first one, result are in Newtons : Force = mass * acceleration, 
mass = 10 kg
acceleration = g = 9.81
Force = 10*9.81 = 98.1 N
For the second one idk !

**Reply by MeetWq** [reply]
But the mass I set is 1kg.

**Reply by ShuffleWire** [reply]
Oh ! did realized that !

**Reply by ShuffleWire** [reply]
Watch out ! There is the lookup table that is setup by default that introduce a x10 !
https://cyberbotics.com/doc/reference/touchsensor?version=master#lookup-table

**Reply by MeetWq** [reply]
OK, get it. But this can easily lead to confusion. Why not set the lookup table to empty by default?

**Reply by omichel** [reply]
I agree, it would be better.
